integration-test/playwright/broker-protection.spec.js

```diff
@@ -381,6 +381,17 @@ test.describe('Broker Protection communications', () => {
             await page.waitForURL(url => url.hash === '#2', { timeout: 2000 })
         })
 
+        test('click multiple targets', async ({ page }, workerInfo) => {
+            const dbp = BrokerProtectionPage.create(page, workerInfo)
+            await dbp.enabled()
+            await dbp.navigatesTo('click-multiple.html')
+            await dbp.receivesAction('click-multiple.json')
+            const response = await dbp.waitForMessage('actionCompleted')
+
+            dbp.isSuccessMessage(response)
+            await page.waitForURL(url => url.hash === '#1-2', { timeout: 2000 })
+        })
+
```

Comment:
@brianhall - thanks! this is a really good test as it's executing every single branch of affected code 👍🏻